statusHandler/view/termuic/termuiRenders/widgetsRender.go

```diff
@@ -146,39 +146,42 @@ func (wr *WidgetsRender) prepareInstanceInfo() {
 	countConsensusAcceptedBlocks := wr.presenter.GetCountConsensusAcceptedBlocks()
 
 	if shardId == uint64(core.MetachainShardId) {
-		consensusInfo = fmt.Sprintf("Count consensus participant: %d | Signed blocks headers: %d", countConsensus, countConsensusAcceptedBlocks)
-
+		consensusInfo = fmt.Sprintf("Validator signed meta blocks: %d | Signed blocks headers: %d", countConsensus, countConsensusAcceptedBlocks)
```

Comment:
This is still not very clear:
consensusInfo = fmt.Sprintf("Validator signed blocks: %d | Included signatures: %d", countConsensus, countConsensusAcceptedBlocks)

Should remove the if.

Reply:
`Validator signed blocks` and `Included signatures` are the same thing. `Validator signed blocks %d | Number accepted: %d` refers to: number of blocks with included signature in the left _and_ number of block accepted with included signature in the right.
Regarding the `if` removal, it cannot be done because for meta we display the number of meta blocks with included signature and the number of headers with the signature included

Reply:
changed `Number` to `Blocks`

Reply:
refactored